modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/kb_template_comments.jsp

```diff
@@ -62,10 +62,10 @@ boolean helpful = BeanParamUtil.getBoolean(kbComment, request, "helpful", true);
 							<c:if test="<%= kbComment != null %>">
 
 								<%
-								request.setAttribute("template_comment.jsp-kb_comment", kbComment);
+								request.setAttribute("kb_template_comment.jsp-kb_comment", kbComment);
```

Comment:
@boton @ambrinchaudhary 

This is ok to change?

Reply:
LGTM. But not tested

Reply:
Just checked you also changed it here, so yes, I'm ok with the change ;-) 
<img width="1308" alt="Screenshot 2022-08-04 at 12 53 00" src="https://user-images.githubusercontent.com/8373764/182830128-36eb1b74-b4c5-40f9-a73b-f92abdba31bc.png">